### DISCUSSION TITLE
How to disable transition/suspense behaviour

### DISCUSSION THREAD

**Discussion by tennox** [solidjs/solid-router#296] [Q&A]
After a few hours of debugging and trying to understand, I found [this issue](https://github.com/solidjs/solid/discussions/1629), and specifically this:

> I also learned that my desired behavior **(instant page navigation + a loading spinner) is considered feeling less responsive than staying on the current page until new data has been loaded**, while displaying some kind of progress indicator. This can be achieved with const isRouting = useIsRouting(); as described [here](https://github.com/solidjs/solid-router#useisrouting).

In [this issue](https://github.com/solidjs/solid-router/issues/60#issuecomment-992145597) @ryansolid says:
> This is intentional. There is a feature called [Transitions](https://www.solidjs.com/tutorial/async_transitions) that are baked into the router that do this hold mechanism and allow with the use of isRouting to indicate stale state. **If you want to show fallback put the Suspense boundary below the route(in your page) is generally the recommended way**.

I think&feel that showing the new page with a loading spinner for certain areas is better UX (in some situations at least) - so I'd like to know how that's possible, as I cannot get it to work, even when throwing `<Suspense>` everywhere - as you can see in my example repository:

https://gitlab.com/TeNNoX/solid-router-repro/

**Comment by Manviel** [comment]
Are there any updates on how to show the skeleton when loading a route in `^0.10.9` ? Previously, in `^0.8.3` , this was achieved by wrapping `Routes` in a `Suspense`, but this doesn't work now.

https://github.com/solidjs/solid-router/issues/102

**Reply by Manviel** [reply]
Hello @ryansolid ,
If this feature is baked, could you provide more concrete example of how Transition should work with latest router package?
Thank you!
